CHANGELOG.md

```diff
@@ -2,5 +2,76 @@
 
 ## Release 2.5.0 (in development)
 
+### Breaking changes
+- Basic authentication has been deprecated in favour of OpenID Connect (OIDC)
+with Dex being the Identity provider of choice.
+This comes with the following:
+    - The Kubernetes API server authentication flag `--basic-auth` will be
+      removed and henceforth no longer be used in favour of `--oidc-issuer-url`.
+    - Fresh installations of MetalK8s will come with 2 Dex pods in a new
+      namespace `metalk8s-auth`
+    - Complete re-brand of the MetalK8s UI
+    - Authentication and access to MetalK8s UI and Grafana services will
+      henceforth implement OIDC based authentication
```

Comment:
Let's focus on user facing changes in here, especially on what is breaking:

```suggestion
- Basic authentication has been deprecated in favour of OpenID Connect (OIDC)
  with Dex being deployed as a local Identity Provider, used by Kubernetes
  API and Grafana.
  
  This implies:
    - The existing users defined for Kubernetes API Basic Auth (in
      `/etc/htpasswd`) and for the Grafana admin will become unusable
    - A default admin user will be created in Dex, with the new
      credentials `admin@metalk8s.invalid`:`password`
    - Procedure to edit and add users can now be found
      [here](link to the docs page, maybe on RTD)
```

Reply:
Done